src/AppInstallerCLICore/Workflows/ImportExportFlow.cpp

```diff
@@ -309,7 +309,8 @@ namespace AppInstaller::CLI::Workflow
                         context.Reporter.Info() << Resource::String::Cancelled << std::endl;
                         return;
                     }
-                    else if (searchContext.GetTerminationHR() == APPINSTALLER_CLI_ERROR_UPDATE_NOT_APPLICABLE)

```

Comment:
>searchContext.GetTerminationHR() [](http://example.com/codeflow?start=29&length=32)

Nit: should this be assigned to a variable so that it is not called 3 times within this if-else block.